src/styles/modules/Title.scss

```diff
@@ -0,0 +1,5 @@
+.app__title {
```

Comment:
i think, that we could avoid here the block statement and name this component (as it is totally separate component now) just title ;)